### ISSUE TITLE
Access to the collection object in the DSL

### ISSUE THREAD

**Issue by shaundrong** [randym/activeadmin-axlsx#5]
Any chance we can get access to the collection object in the DSL vs rebuilding the query via model as shown? Params don't appear to be available in the DSL so rebuilding the users query will be kind of a pain.
Maybe something like this?
lib/active_admin/axlsx/builder.rb

``` ruby
      def sheet_collection
        @sheet_collection
      end

      def sheet_collection=(collection)
        @sheet_collection = collection
      end

      def serialize(collection)
        self.sheet_collection = collection
        apply_filter @before_filter
        export_collection(collection)
        apply_filter @after_filter
        to_stream
      end
```

**Comment by randym** [comment]
Hi @shaundrong 

How about passing it back out in the before and after filers?

**Comment by randym** [comment]
@shaundrong 

Looking into this deeper, I have to confess that I jumped to conclusions about your intent.

I am happy to provide the collection object on the builder so that 'collection' is available in your before and after filters. (this will be been released with 2.1.2 today)

However, your commends regarding Param and user query need some clarification.

**Comment by shaundrong** [comment]
@randym 

Nope 2.1.2 solution was perfect. I just wanted to avoid having to recreate the active admin resource collection with all of its metasearch filters with a class fetch, as your example shows. I was just adding I would have jump through extra hoops to get the active admin params inside axlsx dsl.
